extensions/terminalfinder/package-lock.json

```diff
@@ -1,6474 +1,138 @@
 {
+  "$schema": "https://www.raycast.com/schemas/extension.json",
```

Comment:
Could you move this to the `package.json` file? It looks like it's in the wrong spot.

Reply:
Although I didnt personally add it, but sure ill add it.

Reply:
I have removed it from `package-lock.json` although it was present in `package.json` so no changes there

Reply:
Sorry if my previous explanation wasn't clear enough. The `package-lock.json` file is created by the package manager, but it also includes the contents of `package.json` now.
You should add command definitions to `package.json`.

Reply:
oh mymy, how did that happen. sorry ill edit that!

Reply:
Fixed